dashboard/cron.py

```diff
@@ -681,7 +683,7 @@ def do(self) -> str:
                     exception_in_run = True
 
         logger.info("** informational")
-        status += self.update_unizin_metadata()
+        # status += self.update_unizin_metadata()
```

Comment:
make sure this line will be uncommented once the metadata query is working

Reply:
Added this as a task on the top. For sure it needs to be fixed.